Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Material/ShaderCollection.h

```diff
@@ -50,11 +50,28 @@ namespace AZ
                 //! Required for use in containers; not meant to be called directly.
                 Item();
 
+                enum class DrawItemType : uint32_t
+                {
+                    Raster = 0, // the shader expects a default draw item for a rasterpass (default)
+                    Deferred, // the shader expects a fullscreen draw item for a deferred rendering pass
+                    RayTracing, // the shader expects shader libraries for a raytracing shader table
+                    None, // no draw-item should be created
```

Comment:
Why do we need `None`? Based on the description that no draw item should be created, it sounds like you should just not create one rather than use None?